contrib/automation_tests/core/orbit_e2e.py

```diff
@@ -109,6 +111,14 @@ def find_combo_box_item(self, text: str, raise_on_failure=True):
                                  parent=self.suite.application.top_window(),
                                  raise_on_failure=raise_on_failure)
 
+    def orbit_path(self):
+        """
+        Returns the root dir of the running "Orbit.exe". Throws an exception if there is no "Orbit.exe" running.
+        """
+        running_processes = pywinauto.application.process_get_modules()
+        orbit_exe_path = [path for pid, path, cmd in running_processes if "Orbit.exe" in path].pop()
```

Comment:
My Python is not very fresh: can `_` be used for `pid` and `cmd`?

Reply:
I think we have the same delemma as in c++ here. Adjusted it to match our c++ conventions.

Reply:
This is obsolete now and also I would have been fine with all three versions, but this is very different than C++: in Python variable declarations work very differently and `_` is widely considered the throw-away variable. I was just wondering whether this would work in this specific case.